src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -168,13 +168,11 @@ public void clickCollapseExpandButtonAndWaitForPanelsToExpand() {
     public void expandPanels() {
         if (isElementPresent("collapse-panels-button")) {
             clickCollapseExpandButtonAndWaitForPanelsToExpand();
-            waitForPanelsToExpand();
```

Comment:
Do check where this method is used before modifying it. This causes a change in behaviour in an unrelated test.